Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    [SerializeField]
+    private string scenetoLoad;
+
+    private bool isPlayerInTrigger;
+
+    // cant use this bc it triggers the door twice, bc the player has two colliders
```

Comment:
I'd save this in a document elsewhere if it isn't used anymore, it might be useful for later.

Reply:
I took it out now.